src/custom-element.js

```diff
@@ -381,6 +381,9 @@ export function createAmpElementProto(win, name, opt_implementationClass) {
      * @private {boolean|undefined}
      */
     this.isInTemplate_ = undefined;
+
+    /** @private {boolean} */
+    this.hadLoadingError_ = false;
```

Comment:
Good point. Looking into it more, an element can get have `#layoutCallback` multiple times if it defines a `#isRelayoutNeeded`. So ignore that part. 😬